osu.Game/Screens/Play/PlayerSettings/VisualSettings.cs

```diff
@@ -39,12 +41,21 @@ public VisualSettings()
                     DisplayAsPercentage = true
                 },
                 new OsuSpriteText
+                {
+                    Text = "Playfieldd area dim:"
```

Comment:
if this does indeed stay
```suggestion
                    Text = "Playfield area dim:"
```

Reply:
This should not be here. These settings are meant to be setting you regularly change. Which this is not.